src/test/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/request/RaptorRoutingRequestTransitDataCreatorTest.java

```diff
@@ -113,9 +116,44 @@ private TripTimes createTripTimesForTest() {
     stopTime1.setDepartureTime(0);
     stopTime2.setArrivalTime(7200);
 
-    return new TripTimes(new Trip(new FeedScopedId("Test", "Test")),
-        Arrays.asList(stopTime1, stopTime2),
-        new Deduplicator()
+    return new TripTimes(
+            new Trip(new FeedScopedId("Test", "Test")),
+            Arrays.asList(stopTime1, stopTime2),
+            new Deduplicator()
     );
   }
+
+  /**
+   * Utility function to create bear minimum of valid StopTime with no interesting attributes
+   *
+   * @return StopTime instance
+   */
+  private static StopTime createStopTime() {
+    var st = new StopTime();
+    st.setStop(Stop.stopForTest("Stop:1", 0.0, 0.0));
+    return st;
+  }
+
+  /**
+   * Utility class that does nothing, used just to avoid null value on filter
+   */
+  public static class TransitDataProviderFilterTest implements TransitDataProviderFilter {
```

Comment:
If this inner class is privet, then make it private, if not put it in a separate file - We do not allow `public static`, `protected static` or package local `static` inner classes.